be/src/runtime/buffered_tuple_stream2.h

```diff
@@ -257,6 +257,11 @@ class BufferedTupleStream2 {
     bool has_tuple_footprint() const {
         return _fixed_tuple_row_size > 0 || !_string_slots.empty() || _nullable_tuple;
     }
+    /// Returns true if the row consumes any memory. If false, the stream only needs to
+    /// store the count of rows.
+    bool row_consumes_memory() const {
```

Comment:
**I think the implementation isn't consistent with the method comment and name.**

Only if there has one tuple. _fixed_tuple_row_size will large than zero. 
```
    for (int i = 0; i < _desc.tuple_descriptors().size(); ++i) {
        const TupleDescriptor* tuple_desc = _desc.tuple_descriptors()[i];
        const int tuple_byte_size = tuple_desc->byte_size();
        _fixed_tuple_row_size += tuple_byte_size;
        if (!tuple_desc->string_slots().empty()) {
            _string_slots.push_back(make_pair(i, tuple_desc->string_slots()));
        }
        // if (!tuple_desc->collection_slots().empty()) {
        //     _collection_slots.push_back(make_pair(i, tuple_desc->collection_slots()));
        // }
    }
```

Reply:
I will change the fun name